src/it/saxon/src/test/java/com/jcabi/saxon/SaxonSampleTest.java

```diff
@@ -91,7 +95,13 @@ public Void call() throws Exception {
                 );
                 return null;
             }
-        } .call();
+        };
+        final ExecutorService executorService = Executors.newFixedThreadPool(5);
+        for (int count = zero; count < fifty; count = count + 1) {
```

Comment:
@longtimeago if we do
```
for (...) {
    callables.add(
         new Callable<..>(){...};
     )
}
```
we'll get checkstyle warnings AvoidInstantiating objectsInLoops